src/WeatherRequest.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
```

Comment:
Import` React` as well just to be safe. There could be side effects writing JSX without it.

Reply:
I'm getting an error if you `import React` without explicitly using it in the code. I believe the import isn't necessary anymore after the release of React 17 and the change included support for earlier React versions.

From [react.js.org](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html): 
> Because the new JSX transform will automatically import the necessary react/jsx-runtime functions, React will no longer need to be in scope when you use JSX.

Reply:
Sounds good, I didn't know that!